lib/watcher.js

```diff
@@ -18,6 +19,7 @@ function rethrowAsync(err) {
 
 const MIN_DEBOUNCE_DELAY = 10;
 const INITIAL_DEBOUNCE_DELAY = 100;
+const END_MESSAGE = chalk.gray('Type `r` and press enter to rerun tests\nType `u` and press enter to update snapshots');
```

Comment:
I think it'd be good to write another `\n` at the end. That way if the tests re-rerun (because you change files) there is an empty line before the test run separator.